esmvalcore/cmor/_fixes/cmip6/mpi_esm1_2_hr.py

```diff
@@ -0,0 +1,83 @@
+"""Fixes for MPI-ESM1-2-HR model."""
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
```

Comment:
I think it would be a good idea to change these functions for adding auxiliary coordinates so they check if the coordinate is already present and do nothing if this is the case. That way we will avoid the situation that ESMValTool will crash as soon as corrected data is released.

Reply:
#372

Reply:
Yes, that's a good example, but it would be better to do it in the function, to avoid needing this code that checks if the coordinate exists in every fix.

Reply:
Yeah, I think so too. I only wanted to mention this PR because the code can be used without changes in the shared functions.